doc/troubleshooting.md

```diff
@@ -0,0 +1,31 @@
+[<- Back to Index](index.md)
+
+# Troubleshooting FAQ
+
```

Comment:
I would like a section for the following as well, but you can put at the bottom if you want.

- Is this an LMS? Does it store content?

No, a Learning Record Store (LRS) does not host, distribute, or author learning content, and it does not serve as a content repository.  The role of an LRS is strictly to receive, validate, store, and make available the xAPI statements and other learning data produced by Learning Record Providers (LRPs) such as an LMS.

Reply:
Since it's not necessarily a technical question, I put it under the overview [here](https://github.com/yetanalytics/lrsql/blob/faq-section/doc/overview.md#what-is-sql-lrs-not).

Reply:
After discussion on Slack, I created a General Questions page and put that question [here](https://github.com/yetanalytics/lrsql/blob/faq-section/doc/general_faq.md#is-sql-lrs-a-learning-record-provider-lrp-learning-management-system-lms-or-content-store).